sktime/forecasting/compose/tests/test_reduce_global.py

```diff
@@ -175,3 +184,54 @@ def test_list_reduction(y, index_names):
     forecaster2.fit(y, fh=[1, 2])
     y_pred = forecaster2.predict(fh=[1, 2, 12])
     check_eval(y_pred.index.names, index_names)
+
+
+def test_equality_transfo_nontranso():
+    """Test that recursive reducers return same results for global / local forecasts."""
+    ensemblealgos = [
```

Comment:
Maybe worth putting this into a fixture for better readability, sth like `@pytest.mark.parametrize("regressor", [
        RandomForestRegressor(),
        GradientBoostingRegressor(),
        HistGradientBoostingRegressor(),
    ])`

Reply:
+1